include/dpp/channel.h

```diff
@@ -78,7 +80,19 @@ enum channel_flags : uint8_t {
  */
 enum discord_channel_flags : uint8_t {
 	/// Thread pinned in a forum (type 15) channel
-	dc_pinned_thread = 0b00000001,
+	dc_pinned_thread = 1 << 1,
+	/// whether a tag is required to be specified when creating a thread in a forum channel. Tags are specified in the thread::applied_tags field.
+	dc_require_tag =   1 << 4,
```

Comment:
this flag isn't properly implemented yet, as it needs the channel_flags to be uint16. It is there, but nothing more

Reply:
maybe someone else can do it